Explorer/Assets/Scripts/MVC/ViewDependencies/ViewDependencies.cs

```diff
@@ -0,0 +1,38 @@
+﻿using DCL.Clipboard;
+using DCL.Input;
+using DCL.Multiplayer.Connections.RoomHubs;
+using DCL.Profiles;
+using DCL.UI.Profiles.Helpers;
+
+namespace MVC
+{
+    /// <summary>
+    ///     A set of references to the only systems and managers a view can use directly, without the need of a controller.
+    ///     These should not be able to change the state of the game in a meaningful way, but allow for easier access to certain functionalities needed to display data.
+    /// </summary>
+    public class ViewDependencies
+    {
+        public readonly DCLInput DclInput;
+        public readonly IEventSystem EventSystem;
+        public readonly MVCManagerMenusAccessFacade GlobalUIViews;
+        public readonly IClipboardManager ClipboardManager;
+        public readonly ICursor Cursor;
+        public readonly IProfileCache ProfileCache;
```

Comment:
If you want to have certain getter functions in views - it's not a problem but exposing the whole contract questions the difference between controllers and views and only the former should provoke changes on the model level. Consider introducing a separate getter contract for all "model layer" dependencies you'd like to expose to views